src/logic/contracts/safeContracts.ts

```diff
@@ -54,7 +54,7 @@ const createMasterCopies = async () => {
   const accounts = await web3.eth.getAccounts()
   const userAccount = accounts[0]
 
-  const ProxyFactory = getCreateProxyFactoryContract(web3)
+  const ProxyFactory = getCreateProxyFactoryContract(web3, 4447)
```

Comment:
What is this number?

Reply:
This is a random network id so typescript doesn't complain, this function is not used anymore (to be replaced). It was used only in tests, but we plan a complete rewrite of a test suite so it doesn't use ganache (ganache is not supported by transaction-history-service). So this change doesn't affect anything